.gitignore

```
@@ -15,3 +15,7 @@
 
 # acpanel
 \[admin\]/acpanel/settings.xml
+
+# editor
+\editor_dump/
+\editor_test/
```

```suggestion
editor_dump/
editor_test/

```
The backslash was escaping `[` it's not required at the start of every line

---

done